File: bnctl-banking-android/app/src/main/java/tl/bnctl/banking/data/transfers/model/PendingTransfer.kt
package tl.bnctl.banking.data.transfers.model

data class PendingTransfer(
    var transferId: String,
    var description: String,
    var beneficiary: String,
    var beneficiaryName: String,
    var beneficiaryBankName: String,
    var beneficiaryBankSwiftCode: String,
    var currency: String,
    var amount: String,
    var sourceAccount: String,
    var issuerName: String,
    var destinationAccount: String,
    var status: String,
    var transactionType: String,
    var isSelected: Boolean = false,
    var numberOfDocuments: Int
)

File: banking-ios copy/Shared/Model/Transfer/PendingTransfer.swift
//
//  PendingTransfer.swift
//  Allianz (iOS)
//
//  Created by Evgeniy Raev on 26.11.21.
//

import Foundation

struct PendingTransfer:Codable, Identifiable, Hashable  {
    let transferId:String
    let description:String?
    let additionalDescription:String?
    let beneficiaryBankName:String?
    let beneficiaryName:String?
    let beneficiary:String?
    let beneficiaryBankSwiftCode:String?
    let currency:String
    let amount:Double
    let sourceAccount:String
    let issuerName:String?
    let destinationAccount:String?
    let status:String
    let transactionType:String
    let numberOfDocuments: Int?
    
    var id: String { self.transferId }
}

